daras_ai_v2/gooey_builder.py

```
@@ -237,14 +291,27 @@ class GooeyBuilderSendMessage(pydantic.BaseModel):
+def submit_gooey_builder_message(
+    request: fastapi.Request, body: GooeyBuilderSendMessage
+) -> str:
     from daras_ai_v2.workflow_url_input import url_to_runs
     from functions.gooey_builder_tools import insert_gooey_builder_variables
 
     # inline import to avoid a circular dependency with routers.ask_gooey_new
     from routers.ask_gooey_new import get_gooey_builder_run_url
 
-    workspace = get_current_workspace(request.user, request.session)
+    builder_run_url = body.builder_run_url or get_default_builder_pr().get_app_url()
+    builder_page_cls, builder_sr, builder_pr = url_to_runs(builder_run_url)
```

Ruff (0.16.3)

[warning] Unpacked variable `builder_page_cls` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)